src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -1101,6 +1202,82 @@ components:
         - generated
         - printed
 
+    AedSearchSummaryResponse:
+      title: AedSearchSummaryResponse
+      description: Response containing the meta data and results of searching for Anonymous Elector document summaries
+      type: object
+      properties:
+        results:
+          type: array
+          description: Array of anonymous elector document search summaries that matches the search criteria
+          items:
+            $ref: '#/components/schemas/AedSearchSummary'
+      required:
+        - results
+
+    AedSearchSummary:
+      title: AedSearchSummary
+      description: An object containing summary information of an each Anonymous Elector document summary
+      type: object
+      properties:
+        gssCode:
+          type: string
+          description: The relevant GSS code for the local authority.
+          example: 'E09000007'
+        sourceReference:
+          type: string
+          pattern: '^[a-fA-F\d]{24}$'
+          description: |
+            A 24 character hex string. 
+            The unique identifier of an Anonymous elector document application.
+          example: 1f0f76a9a66f438b9bb33070
+        applicationReference:
+          type: string
+          description: The reference number of an Anonymous elector document application
+          example: A3JSZC4CRH
+        certificateNumber:
+          type: string
+          description: The unique identifier of the Anonymous elector document. A 20 character alphanumeric string.
+          pattern: '^[0-9AC-HJ-NP-RT-Z]{20}$'
+          example: DEV1W0XDH1368LWE1J40
+        electoralRollNumber:
+          type: string
+          description: Electoral roll number printed on the AED. There is no specific format and they can vary across EROs.
+          maxLength: 25
+        firstName:
+          type: string
+          description: The applicant's first name
+          example: Fred
+        surname:
+          type: string
+          description: The applicant's surname
+          example: Blogs
+        postcode:
+          type: string
+          description: The applicant's postcode
+          example: AA1 1AA
+        issueDate:
```

Comment:
I know the UI wants to render this as "Printed Date" (issue == printed), but I'm wondering about the date aspect of it ... 🤔 
Presumably we actually hold it in the DB as a date time ? Should we be returning it as a date time (which will make sorting easier as well), and then the UI can decide for themselves how they want to render the value